### PR TITLE
chore: update Travis CI setup to reflect the config of n98-magerun

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,45 +1,103 @@
 language: php
 sudo: false
+dist: precise
```
Comment (vdloo) [inline .travis.yml:3]: we should upgrade this to xenial some time in the future

**Review by vdloo — APPROVED** [review]
very nice, thanks @DanielRuf

File: .travis.yml (hunk 1 above)
Comment (DanielRuf) [reply .travis.yml:3]: See https://docs.travis-ci.com/user/reference/overview/#Virtualisation-Environment-vs-Operating-System for an overview of the environments that Travis provides by default.

Comment (vdloo) [reply .travis.yml:3]: ah no xenial yet, then precise is definitely fine :+1:

**Pull request merged**